src/instructlab/model/train.py

```diff
@@ -312,13 +404,15 @@ def train(
     if four_bit_quant and device != "cuda":
         ctx.fail("'--4-bit-quant' option requires '--device=cuda'")
 
-    effective_data_dir = Path(data_path if data_path else DEFAULTS.DATASETS_DIR)
-    train_file = Path(effective_data_dir / "train_gen.jsonl")
-    test_file = Path(effective_data_dir / "test_gen.jsonl")
+    effective_data_dir: pathlib.Path = Path(
```

Comment:
(Curious) why would we need to list the typing hint here when it's clear from the code it's a `Path` (`mypy` should be able to deduce it already.)

Reply:
muscle-memory tbh. will remove.